bincommon/commonflags.go

```diff
@@ -132,6 +133,22 @@ func SharedMain(usage func(io.Writer, ...interface{})) {
 		"Size of the buffer (max data size) for the optimized http client in `kbytes`")
 	flag.BoolVar(&fhttp.CheckConnectionClosedHeader, "httpccch", fhttp.CheckConnectionClosedHeader,
 		"Check for Connection: Close Header")
+	// FlagResolveIPType indicates which IP types to resolve.
+	// With round robin resolution now the default, you are likely to get ipv6 which may not work if
+	// use both type (`ip`). In particular some test environments like the CI do have ipv6
+	// for localhost but fail to connect. So we made the default ip4 only.
+	dflag.Flag("resolve-ip-type", fnet.FlagResolveIPType)
+	// FlagResolveMethod decides which method to use when multiple ips are returned for a given name
+	// default assumes one gets all the ips in the first call and does round robin across these.
+	// first just picks the first answer, rr rounds robin on each answer.
+	dflag.Flag("dns-method", fnet.FlagResolveMethod)
+	dflag.Flag("echo-server-default-params", fhttp.DefaultEchoServerParams)
+	dflag.FlagBool("proxy-all-headers", fhttp.Fetch2CopiesAllHeader)
+	dflag.Flag("server-idle-timeout", fhttp.ServerIdleTimeout)
+	// MaxDelay is the maximum delay allowed for the echoserver responses.
+	// It is a dynamic flag with default value of 1.5s so we can test the default 1s timeout in envoy.
+	dflag.Flag("max-echo-delay", fhttp.MaxDelay)
+	dynloglevel.LoggerFlagSetup()
```

Comment:
Initial thought is; what/why is this? (haven't checked it locally and followed the reference, but might be nice to comment).

Reply:
let me add a comment

it's https://pkg.go.dev/fortio.org/dflag/dynloglevel#LoggerFlagSetup